### PR TITLE
feature: Add apply button and fire call only when clicked

### PR DESCRIPTION
**Pull request by ManyRios**
## Describe your changes
Add an apply button at the bottom of the filter. Fire the call with the correctly selected statuses after the save button has been clicked.

https://github.com/stakwork/sphinx-tribes-frontend/assets/34518489/84b4fee9-e2dc-4caf-839d-a669cfed5d2c

image

Screenshot_2

## Issue ticket number and link
 https://github.com/stakwork/sphinx-tribes-frontend/issues/347 

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [x] I have performed a self-review of my code
- [x] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [x] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by Vayras** [comment]
hey! @ManyRios can you add a tad bit more top margin for apply button? and maybe move the filter component bit to the right? rest is working as intended
Screenshot 2024-03-02 at 12 16 59 PM

**Comment by ManyRios** [comment]
> hey! @ManyRios can you add a tad bit more top margin for apply button? and maybe move the status component bit to the right? rest is working as intended Screenshot 2024-03-02 at 12 16 59 PM

yeah, the top margin could be better

File: src/pages/superadmin/index.tsx [hunk 1]
```diff
@@ -98,9 +99,20 @@ export const SuperAdmin = () => {
     }
   }, [main, startDate, endDate, checkboxIdToSelectedMap, sortOrder, currentPage]);
 
+  useEffect(() => {
+    if (search) {
```
[the remark this answers is not in the document]
Comment (ManyRios) [reply src/pages/superadmin/index.tsx]: hey @elraphty please take another look

**Comment by elraphty** [comment]
The same issue @ManyRios the Table state change is only triggered by the apply button, which should be so, the Sort and Date change should also trigger the table update. 

Loom video

https://www.loom.com/share/35c7cc34cbd74251bc1353c8bde50fc0

**Comment by ManyRios** [comment]
> The same issue @ManyRios the Table state change is only triggered by the apply button, which should be so, the Sort and Date change should also trigger the table update.
> 
> Loom video
> 
> https://www.loom.com/share/35c7cc34cbd74251bc1353c8bde50fc0

hmmm you're right... Fixing this right now. I hadn't checked this

**Comment by ManyRios** [comment]
@elraphty 

https://github.com/stakwork/sphinx-tribes-frontend/assets/34518489/180c524a-4a9c-424a-a4ac-71b6bd1cc45d

**Pull request merged**